src/utils/format.js

```diff
@@ -226,3 +226,40 @@ export function formatHexAddress(address: string) {
 
   return t('ellipsedMiddleString', { stringStart: address.slice(0, 6), stringEnd: address.slice(-6) });
 }
+
+export function formatExchangeRate(
+  rate: ?BigNumber | number,
+  fromSymbol: string,
+  toSymbol: string,
+  reverse: boolean = false,
+) {
+  rate = wrapBigNumber(rate);
+  if (!rate || !rate.isFinite()) return null;
+
+  if (reverse) {
+    rate = BigNumber(1).dividedBy(rate);
+    return formatExchangeRate(rate, toSymbol, fromSymbol);
+  }
+
+  const rateString = formatExchangeRateWithoutSymbol(rate);
+  return t('exchangeRate', { rate: rateString, fromSymbol, toSymbol });
+}
+
+export function formatExchangeRateWithoutSymbol(rate: ?BigNumber | number) {
```

Comment:
Weird, I think I've seen this util before, but can't find anything 🤔

Reply:
Maybe there was some older one, but I intend `utils/format` to contain modern replacement for old formatting functions.